class-wxr-importer.php

```diff
@@ -337,9 +339,15 @@ public function import( $file ) {
 
 		// Reset other variables
 		$this->base_url = '';
+		$element_count = 0;
 
 		// Start parsing!
 		while ( $reader->read() ) {
+			$element_count++;
+			if ( $element_count < $this->current_node ) {
+				continue;
+			}
+			$this->current_node = $element_count;
```

Comment:
This should be _after_ the `::ELEMENT` check, otherwise you could resume half-way through an attribute.